### PR TITLE
Release v1.67.0

### PR DESCRIPTION
**Pull request by YashdalfTheGray**
## 1.67.0

* Bug - Don't log errors on instances not using GMSA [#3489](https://github.com/aws/amazon-ecs-agent/pull/3489)
* Enhancement - Update packaging Readme files with updated instructions to build init files [#3490](https://github.com/aws/amazon-ecs-agent/pull/3490)
* Bug - Fix unit tests for cgroup v2 [#3491](https://github.com/aws/amazon-ecs-agent/pull/3491)
* Enhancement - Update readme for ECS_SELINUX_CAPABLE to clarify Z-mode mount only and limited support [#3496](https://github.com/aws/amazon-ecs-agent/pull/3496)
* Bug - Fix agent short hash version bug [#3497](https://github.com/aws/amazon-ecs-agent/pull/3497)
* Bug - Use Ubuntu 20.04 for linux GH Unit tests [#3501](https://github.com/aws/amazon-ecs-agent/pull/3501)
YashdalfTheGray marked this conversation as resolved.
* Feature - Container port range mapping [#3506](https://github.com/aws/amazon-ecs-agent/pull/3506)
### Licensing

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,13 @@
 # Changelog
+
+## 1.67.0
+* Bug - Don't log errors on instances not using GMSA [#3489](https://github.com/aws/amazon-ecs-agent/pull/3489)
+* Enhancement - Update packaging Readme files with updated instructions to build init files [#3490](https://github.com/aws/amazon-ecs-agent/pull/3490)
+* Enhancement - Update readme for ECS_SELINUX_CAPABLE to clarify Z-mode mount only and limited support  [#3496](https://github.com/aws/amazon-ecs-agent/pull/3496)
+* Bug - Fix agent short hash version bug [#3497](https://github.com/aws/amazon-ecs-agent/pull/3497)
+* Bug - Use Ubuntu 20.04 for linux GH Unit tests [#3501](https://github.com/aws/amazon-ecs-agent/pull/3501)
```
Comment (singholt) [inline CHANGELOG.md:8]: superseded by https://github.com/aws/amazon-ecs-agent/pull/3491 ?

File: agent/version/version.go [hunk 2]
```diff
@@ -22,10 +22,10 @@ package version
...
 // Version is the version of the Agent
-const Version = "1.66.2"
+const Version = "1.67.0"
 
 // GitDirty indicates the cleanliness of the git repo when this agent was built
 const GitDirty = true
 
 // GitShortHash is the short hash of this agent build
-const GitShortHash = "6e8c9a51"
+const GitShortHash = "d49ed882"
```
Comment (singholt) [inline agent/version/version.go:31]: should this be the last commit on dev? `https://github.com/aws/amazon-ecs-agent/commits/dev` i.e. `d1fa74b5`

File: CHANGELOG.md (hunk 1 above)
Comment (chienhanlin) [reply CHANGELOG.md:8]: or combine both bug fix in an entry?
Bug - Use Ubuntu 20.04 for linux GH Unit tests [#3501](https://github.com/aws/amazon-ecs-agent/pull/3501) and Fix unit tests for cgroup v2 [#3491](https://github.com/aws/amazon-ecs-agent/pull/3491)

**Review by singholt — DISMISSED** [review]
🚢 
the PR description may need an update.

**Review by chienhanlin — APPROVED** [review]

**Comment by prateekchaudhry** [comment]
nit: #3491 also reverses the changes introduced in #3501 by using ```ubuntu-latest``` again, so we might include this pr in notes as well

**Review by Realmonia — APPROVED** [review]

**Review by prateekchaudhry — APPROVED** [review]

**Pull request merged**